src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -39,6 +39,10 @@
 MAX_TAG_VALUE_LENGTH = 200
 
 ACCEPTED_METRIC_TYPES = {"s", "c", "d"}  # set, counter, distribution
+MRI_RE_PATTERN = re.compile("^([c|s|d|g|e]):([a-zA-Z0-9_]+)/.*$")
+
+UseCaseId = str
```

Comment:
might be a nit but, since we now have the `UseCaseID` enum also defined, and this `UseCaseId` is just a alias for a string, can we call it something slightly different as to not get confused?

Reply:
I think I will rename it `UseCase`? I think keeping it as a string is probably the better call since we need to parse it in and out of jsons